utils/use_require.js

```diff
@@ -97,11 +97,12 @@ var make_lib_files = function (import_format, source_maps, with_app_dir) {
     const babel = require('babel-core');
 
     var in_path;
+    var out_path_base;
```

Comment:
Yup. But it obscures code. It's sort of good practice that, if using vars, they all should be declare on top of the scope.
Anyway. This will change to `let` so it will be scoped.